### PR TITLE
add lint prettier in Ci

### PR DESCRIPTION
**Pull request by chenshuai2144**
# New feature added

add a prettier lint in CI.
Since the prettier is only executed in pre-commit, there is a phenomenon of community bypass. So put it in ci.

### Impact on users and possible risks

Increase in ci has no effect on existing functions

### CODE REVIEW

**Comment by afc163** [comment]
不能直接在 ci 环境换一个 eslint-config 的配置么？

**Comment by chenshuai2144** [comment]
为什么要有区别呢？

**Comment by afc163** [comment]
就是自己写 prettier 的 diff 检查逻辑感觉很奇怪，是不是可以找个社区轮子。或者提成一个独立的社区轮子。

**Comment by chenshuai2144** [comment]
prettier.check就是为了做这件事

File: scripts/ci-prettier.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      const [options, fileInfo] = resolves;
+      if (fileInfo.ignored) {
+        return;
+      }
+      const input = fs.readFileSync(file, 'utf8');
+      const withParserOptions = {
+        ...options,
+        parser: fileInfo.inferredParser,
+      };
+      const isPrettier = prettier.check(input, withParserOptions);
```
Comment (afc163) [inline scripts/ci-prettier.js]: https://github.com/hexacta/prettier-check

**Pull request merged**